stack.cabal

```diff
@@ -1,5 +1,5 @@
 name:                stack
-version:             0.1.2.1
+version:             0.1.2.9999
```

Comment:
Actually, it's unclear to me by the PVP whether this PR should technically bump the major version number.  It changes an executable rather than a library, and any usage that worked before 
*should* still work.  (Not so if `stack exec` kept the lock though, in which case you could easily construct usages that *required* concurrent execution.)